css-box-model/styles.css

```diff
@@ -0,0 +1,10 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin: 0 12px;
+  padding: 6px 16px;
+  border-radius: 6px;
+  border: none;
```

Comment:
Remove line 9. The button should have a border, using the default styling from the browser

Reply:
![35135746546](https://user-images.githubusercontent.com/122063836/214897449-299d997b-c1a7-43d0-9e96-70c9cd2c6dd4.PNG)